web/ui/dashboard/src/app/private/components/create-transform-function/create-transform-function.component.ts

```diff
@@ -20,15 +20,26 @@ export class CreateTransformFunctionComponent implements OnInit {
 	@ViewChild('payloadEditor') payloadEditor!: MonacoComponent;
 	@ViewChild('functionEditor') functionEditor!: MonacoComponent;
 	@ViewChild('outputEditor') outputEditor!: MonacoComponent;
+	@ViewChild('outputDiffEditor') outputDiffEditor!: MonacoComponent;
 	@Output('subscriptionFunction') subscriptionFunction: EventEmitter<any> = new EventEmitter();
+	tabs = ['output', 'diff'];
+	activeTab = 'output';
 	transformForm: FormGroup = this.formBuilder.group({
 		payload: [null],
 		function: [null]
 	});
 	isTransformFunctionPassed = false;
 	isTestingFunction = false;
-	payload: any;
-	setFunction: any;
+	payload: any = {
+		id: 'Sample-1',
+		name: 'Sample 1',
+		description: 'This is sample data #1'
+	};
+	setFunction = `// Transform only supports one method, and the function must be named transform as provided below
+// An argument e must be passed into the function which represents the payload to be transformed
+function transform(e) {
+    // Transform function here
+}`;
```

Comment:
Please update this guy:
`/*  1. While you can write multiple functions, the main function
    called for your transformation is the transform function.
    
    2. The only argument acceptable in the transform function is
    the payload data.
    
    3. The transform method must return a value.
*/

const transform = (payload) => {
    // Transform function here
}
`

Reply:
@Oluwadaminiola Did you see this?